bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -215,6 +224,41 @@ TEST_F(InstanceAdminIntegrationTest, ListClustersTest) {
   instance_admin_->DeleteInstance(id);
 }
 
+/// @test Verify that InstanceAdmin::UpdateCluster works as expected.
+TEST_F(InstanceAdminIntegrationTest, UpdateClusterTest) {
+  std::string id =
+      "it-" + bigtable::testing::Sample(generator_, 8,
+                                        "abcdefghijklmnopqrstuvwxyz0123456789");
+  bigtable::InstanceId instance_id(id);
+  bigtable::ClusterId cluster_id(id + "-cl");
+
+  auto config =
+      bigtable::ClusterConfig("us-central1-f", 0, bigtable::ClusterConfig::HDD);
+
+  auto clusters_before = instance_admin_->ListClusters(instance_id.get());
```

Comment:
This test is failing when running against production because the instance does not exist:

```
[ RUN      ] InstanceAdminIntegrationTest.UpdateClusterTest
unknown file: Failure
C++ exception with description "InstanceAdmin::ListClusters(parent=projects/cloud-cpp-testing-resources): InstanceAdmin::ListClusters(parent=projects/cloud-cpp-testing-resources) [5=NOT_FOUND] - " thrown in the test body.
[  FAILED  ] InstanceAdminIntegrationTest.UpdateClusterTest (298 ms)
```

Clusters do not exist in isolation, an instance is a container for a collection of clusters.  You should create an instance and make a change to the clusters in that instance: update the cluster created with the original instance, or create a new cluster and update that one (the latter is probably overkill).

Reply:
changed the code and created the instance first, created a new cluster and updated newly created one.